packages/widget-next-playground/.env.local

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_VITE_WALLET_CONNECT=5432e3507d41270bee46b7b85bbc2ef8
```

Comment:
Could we simplify this by removing the VITE part, as this is no longer a vite project? 🙂

Reply:
😂 I can't believe I didn't see that. Will remove it 👍